salt/modules/netmiko_mod.py

```diff
@@ -226,10 +226,13 @@ def __virtual__():
             False,
             "The netmiko execution module requires netmiko library to be installed.",
         )
-    if salt.utils.platform.is_proxy() and __opts__["proxy"]["proxytype"] != "netmiko":
+    if (
+        salt.utils.platform.is_proxy()
+        and __opts__["proxy"]["proxytype"] == "deltaproxy"
```

Comment:
This would attempt to load any proxy type that is not deltaproxy. What was the proxytype that it was trying to load? Can we add that to the list of proxytypes to ensure only the supported proxy's are loaded.

Reply:
Actually I am restoring previous functionality, that is, previously it could load any proxy type so long as HAS_NETMIKO was true.  The code that is there now is wrong and it was discussed with @garethgreenaway since he wrote it and what he really was attempting to do, was to ensure that the module load failed with deltaproxy.

There are a long list of uses for netmiko that do not involve the deltaproxy, in this particular case it was with Napaplm when interfacing with a Cisco IOS where it uses the underlying netmiko to communicate with the router, hence proxytype is napalm but the use with netmiko is valid.

The above change was decided upon from discussion with @garethgreenaway as to what was the best way to ensure this was not used with deltaproxy.